src/app/TimedHandler.cpp

```diff
@@ -101,7 +99,7 @@ CHIP_ERROR TimedHandler::OnMessageReceived(Messaging::ExchangeContext * aExchang
 
 void TimedHandler::OnExchangeClosing(Messaging::ExchangeContext *)
 {
-    InteractionModelEngine::GetInstance()->OnTimedInteractionFailed(this);
+    mpImEngine->OnTimedInteractionFailed(this);
```

Comment:
since you are using pointer, you need whether it is null and print the error log?